### ISSUE TITLE
Feature request: system-wide server setting

### ISSUE THREAD

**Issue by edekeijzer** [nextcloud/integration_paperless#25]
Looking at Nextcloud + Paperless as an alternative for big-data solutions, wouldn't it be a good thing if an administrator could pre-fill and/or lock the Paperless server being used? I would now have to instruct my users to fill in http://paperless.internal:8000 there, as there's no line of sight from my Nextcloud installation to Paperless-ngx on the public facing domain name.

**Comment by provokateurin** [comment]
Yes this should be added. For a first version I only implemented the personal settings, but admins should be able to configure this as well.
